source/docs/software/networktables/client-side-program.rst

```diff
@@ -3,47 +3,132 @@
 Creating a client-side program
```

Comment:
The gradle code block should be updated for additional platformas (at least macOS Arm

Reply:
Let's wait to do this since it looks like we're migrating to universal binaries.